server/routes/checkYourAnswers/checkYourAnswers.ts

```diff
@@ -1,15 +1,52 @@
 import { Request, Response } from 'express'
 import { FormError } from '../../@types/template'
 import PlaceOnReportService from '../../services/placeOnReportService'
+import LocationService from '../../services/locationService'
 
-export default class IncidentStatementRoutes {
-  constructor(private readonly placeOnReportService: PlaceOnReportService) {}
+type PageData = {
+  error?: FormError | FormError[]
+}
+
+export default class checkYourAnswersRoutes {
+  constructor(
+    private readonly placeOnReportService: PlaceOnReportService,
+    private readonly locationService: LocationService
+  ) {}
+
+  private renderView = async (req: Request, res: Response, pageData: PageData): Promise<void> => {
+    const { error } = pageData
+    const { prisonerNumber, id } = req.params
+    const { user } = res.locals
+
+    const prisoner = await this.placeOnReportService.getPrisonerDetails(prisonerNumber, user)
```

Comment:
Always best to wrap all the API calls in a `Promis.all` so that the requests are made concurrently.